README.md

```diff
@@ -142,3 +142,9 @@ npm run test-watch
 This also prints the current test coverage.
 
 MacOS users might need to upgrade watchman with `brew install watchman` when experiencing troubles with the watch mode. See this issue for details: https://github.com/facebook/jest/issues/1767
+
+# Releasing
+
+A new version of reactist is published both on npm and GPR whenever a new release on Github is created. A Github Action will automatically perform all the necessary steps.
```

Comment:
```suggestion
A new version of reactist is published both on npm and GitHub Package Registry whenever a new release on GitHub is created. A GitHub Action will automatically perform all the necessary steps.
```

I think the GPR acronym isn't used too often, at least I couldn't find many results when Googling, so it's probably better to be clear 👍